test/test_fakescreen.py

```diff
@@ -370,15 +370,15 @@ def test_hammer_tile(self):
     for i in range(7):
         self.testXclock()
     for i in range(30):
-        old_group = (i + 1) % 4
-        if old_group == 0:
-            name = 'a'
-        elif old_group == 1:
-            name = 'b'
-        elif old_group == 2:
-            name = 'c'
-        elif old_group == 3:
-            name = 'd'
+        # old_group = (i + 1) % 4
```

Comment:
If some variable is not used, that doesn't mean you should comment it out, I'm not going to point them all out, but they all should be uncommented